### PR TITLE
Add directions for getting crash dumps from Task Manager.

### PR DESCRIPTION
**Pull request by aluhrs13**

### CODE REVIEW

**Review by lflores-ms — APPROVED** [review]
👍

File: diagnostics/task_manager_dump.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Crash Dump from Task Manager
+Normally, crash dumps are gathered automatically and available in the app's user data folder (see [Crash Dumps](crash.md)). In some cases, a user might need to manually gather a crash dump, normally for unresponsive processes. To do so:
+
+1. Open Task Manager (Ctrl+Shift+Esc).
+2. Click "Details" on the left menu.
+3. Sort by the "Name" column.
+4. Scroll to find "msedgewebview2.exe"
+5. To find the right msedgewebview2.exe, you can use a combination of:
+    1. The "Package name" column, which will represent the application using WebView2.
```
Comment (vbryh-msft) [inline diagnostics/task_manager_dump.md:9]: my ask manager does not have "package name" column to show

Comment (lflores-ms) [reply diagnostics/task_manager_dump.md:9]: Columns can be enabled/disabled. We might need to add instructions or link.
image

File: diagnostics/task_manager_dump.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Crash Dump from Task Manager
+Normally, crash dumps are gathered automatically and available in the app's user data folder (see [Crash Dumps](crash.md)). In some cases, a user might need to manually gather a crash dump, normally for unresponsive processes. To do so:
+
+1. Open Task Manager (Ctrl+Shift+Esc).
+2. Click "Details" on the left menu.
+3. Sort by the "Name" column.
+4. Scroll to find "msedgewebview2.exe"
+5. To find the right msedgewebview2.exe, you can use a combination of:
+    1. The "Package name" column, which will represent the application using WebView2. (you might need to right-click the column headers and check the box for "Package name")
```
Comment (lflores-ms) [inline diagnostics/task_manager_dump.md]: nit: casing in "you" for column annotation.

File: diagnostics/task_manager_dump.md (hunk 1 above)
Comment (vbryh-msft) [reply diagnostics/task_manager_dump.md]: found by following steps

**Review by vbryh-msft — APPROVED** [review]

**Pull request merged**